modules/ServerAPI/src/server/routes/GetLinkExptQueryTool.coffee

```diff
@@ -12,10 +12,10 @@ exports.getLinkQueryToolForExperiment = (req, resp) ->
 	if tool is 'LiveDesign'
 		getLdUrl = require './CreateLiveDesignLiveReportForACAS.js'
 		username = req.session.passport.user.username
-		getLdUrl.getUrlForNewLiveDesignLiveReportForExperimentInternal req.query.experiment, username, (url) ->
+		getLdUrl.getUrlForNewLiveDesignLiveReportForExperimentInternal req.query.experiment, username, (statusCode, url) ->
 			url = url.replace /(\r\n|\n|\r)/gm,""
 			console.log "generating link: #{url}"
-			resp.status(200).send url
+			resp.status(statusCode).send url
```

Comment:
`url` may be either a url or an error message depending on whether the LR creation was successful or not.

Reply:
Interesting. What are the cases where a LR creation would fail?

Reply:
This ticket ACAS-332 is one. Basically there is a situation where the project restrictions of LD and ACAS doesn't align, so the user can see the experiment in ACAS and request a LR from within ACAS based on their permissions, but they won't be able to view the LR in LD due to their permissions.